src/bokeh/server/views/ws.py

```diff
@@ -23,6 +23,7 @@
 # Standard library imports
 import calendar
 import datetime as dt
+from datetime import timezone
```

Comment:
We don't really want two separate imports of `datetime` here, just one should be sufficient. I'd would be inclined to keep the original import and change all new mentions of `timezone` to `dt.timezone`.

Reply:
@ianthomas23 Got you. Addressed these in the recent commit